p8e-encryption/src/main/kotlin/io/provenance/p8e/encryption/util/ExternalKeyCustodyApi.kt

```diff
@@ -0,0 +1,24 @@
+package io.provenance.p8e.encryption.util
+
+import com.fortanix.sdkms.v1.api.SecurityObjectsApi
+
+/**
+ * A wrapper class for all external key custody apis that are needed.
+ *
+ */
+class ExternalKeyCustodyApi(
```

Comment:
Data class crossed my mind, but thought about future implementation and probably wouldn't make sense, probably need to convert to a factory eventually, but we don't know what the APIs will look like yet.